core/assets/bundles/bundle.properties

```diff
@@ -673,4 +669,113 @@ tutorial.router = Factories need resources to function.\nCreate a router to spli
+block.mechanical-drill.description = A cheap drill. When placed on appropriate tiles, outputs items at a slow pace indefinitely.
+block.pneumatic-drill.description = An improved drill which is faster and able to process harder materials by making use of air pressure.
+block.laser-drill.description = Allows drilling even faster through laser technology, but requires power. Additionally, radioactive thorium can be retrieved with this drill.
+block.blast-drill.description = The ultimate drill. Requires large amounts of power.
+block.water-extractor.description = Extracts water from the ground. Use it when there is no lake nearby.
+block.cultivator.description = Cultivates the soil with water in order to obtain biomatter.
+block.oil-extractor.description = Uses large amounts of power in order to extract oil from sand. Use it when there is no direct source of oil nearby.
+block.dart-ship-pad.description = Leave your current vessel and change into a basic fighter aircraft.\nUse the pad by double tapping while standing on it.
+block.trident-ship-pad.description = Leave your current vessel and change into a reasonably well armored heavy bomber.\nUse the pad by double tapping while standing on it.
+block.javelin-ship-pad.description = Leave your current vessel and change into a strong and fast interceptor with lightning weapons.\nUse the pad by double tapping while standing on it.
+block.glaive-ship-pad.description = Leave your current vessel and change into a large, well-armored gunship.\nUse the pad by double tapping while standing on it.
+block.tau-mech-pad.description = Leave your current vessel and change into a support mech which can heal friendly buildings and units.\nUse the pad by double tapping while standing on it.
+block.delta-mech-pad.description = Leave your current vessel and change into a fast, lightly-armored mech made for hit-and-run attacks.\nUse the pad by double tapping while standing on it.
+block.omega-mech-pad.description = Leave your current vessel and change into a bulky and well-armored mech, made for front-line assaults.\nUse the pad by double tapping while standing on it.
+block.spirit-factory.description = Produces light drones which mine ore and repair blocks.
+block.phantom-factory.description = Produces advanced drone units which are significantly more effective than a spirit drone.
+block.wraith-factory.description = Produces fast, hit-and-run interceptor units.
+block.ghoul-factory.description = Produces heavy carpet bombers.
+block.dagger-factory.description = Produces basic ground units.
+block.titan-factory.description = Produces advanced, armored ground units.
+block.fortress-factory.description = Produces heavy artillery ground units.
+block.revenant-factory.description = Produces heavy laser ground units.
+block.repair-point.description = Periodically repairs units in its vicinity.
```

Comment:
Repair points heal the closest damaged unit in their vicinity, continuously. They're not periodic.